Tests/Functional/ViewHelpers/Be/Security/IsAdministratorViewHelperTest.php

```diff
@@ -11,39 +11,32 @@
 
 namespace JWeiland\WallsIoProxy\Tests\Unit\ViewHelpers\Be;
 
-use Nimut\TestingFramework\TestCase\FunctionalTestCase;
-use Prophecy\PhpUnit\ProphecyTrait;
-use Prophecy\Prophecy\ObjectProphecy;
+use PHPUnit\Framework\MockObject\MockObject;
 use TYPO3\CMS\Core\Authentication\BackendUserAuthentication;
 use TYPO3\CMS\Fluid\View\StandaloneView;
+use TYPO3\TestingFramework\Core\Functional\FunctionalTestCase;
 
 /**
  * Test IsAdministratorViewHelper
  */
 class IsAdministratorViewHelperTest extends FunctionalTestCase
 {
-    use ProphecyTrait;
-
-    /**
-     * @var array
-     */
-    protected $testExtensionsToLoad = [
-        'typo3conf/ext/walls_io_proxy',
+    protected array $testExtensionsToLoad = [
+        'jweiland/walls_io_proxy',
```

Comment:
This is not the packagist name

```suggestion
        'jweiland/walls-io-proxy',
```